integration-tests/openshift-client/src/main/java/io/quarkus/it/openshift/client/Routes.java

```diff
@@ -1,20 +1,23 @@
 package io.quarkus.it.openshift.client;
 
+import javax.inject.Inject;
 import javax.ws.rs.GET;
 import javax.ws.rs.Path;
 import javax.ws.rs.PathParam;
 import javax.ws.rs.core.Response;
 
+import io.fabric8.kubernetes.client.KubernetesClient;
 import io.fabric8.openshift.client.OpenShiftClient;
 
 @Path("/route")
 public class Routes {
 
-    private final OpenShiftClient openshiftClient;
+    // make sure we can inject both aspects of it
+    @Inject
+    KubernetesClient kubernetesClient;
 
-    public Routes(OpenShiftClient openshiftClient) {
-        this.openshiftClient = openshiftClient;
-    }
+    @Inject
+    OpenShiftClient openshiftClient;
```

Comment:
Can you add a test that `assertSame(kubernetesClient, openshiftClient)`?

Reply:
Done.